lab_1/main.py

```diff
@@ -69,7 +105,24 @@ def get_concordance(tokens: list, word: str, left_context_size: int, right_conte
     right_context_size = 3
     --> [['man', 'is', 'happy', 'the', 'dog', 'is'], ['dog', 'is', 'happy', 'but', 'the', 'cat']]
     """
-    pass
+    concordance = []
+    if isinstance(tokens, list) and isinstance(word, str) \
```

Comment:
Usually, there are checks for arguments and a function bode - on the same level.
Try to separate them and get rid of nesting.